wasi-hello-world/src/main.rs

```diff
@@ -0,0 +1,15 @@
+pub const DEBUG_ECHO_DEVICE_NAME_LABEL: &str = "DEBUG_ECHO_DESCRIPTION";
+
+use std::env;
+
+fn main() {
+    let device_name = match env::var(DEBUG_ECHO_DEVICE_NAME_LABEL) {
+        Ok(n) => n,
+        Err(_e) => "debugEcho".to_string(),
```

Comment:
Should this error? how did you choose `debugEcho`?

Reply:
Oh, yes it should error, will update it and push a new version. Think I got confused while coding it. Thanks

Reply:
Think I remembered what happened. Instead of using the env var by normal I have used the same code as the proxy and haven't updated it properly hahaha 🙈